add metrics-server v0.6.1

ggpaue opened the pull request:
## What this PR does / why we need it
<!--
add metrics-server v0.6.1
-->

## Which issue(s) this PR fixes
<!--
Usage: `Fixes #<issue number>`, or `Fixes (paste link of issue)`.
-->
Fixes: #

## Describe testing done for PR
<!--
Manually tested package on cluster
-->

## Special notes for your reviewer
<!--
Add any things that reviewers should be aware of as they review
your PR.

Example: Please verify how I handled foo aligns with overall plan.
-->

ggpaue committed:
add metrics-server v0.6.1

seemiller reviewed (changes requested):
I think the `test.yaml` file is extraneous, and it is causing an error with the overlays.
```
cd addons/packages/metrics-server/0.6.1/bundle/config
ytt -f .
```
yields the error:
```
ytt: Error: Overlaying (in following order: overlays/overlay-apiservice.yaml, overlays/overlay-deployment.yaml, overlays/overlay-namespace.yaml, overlays/update-strategy-overlay.yaml):
  Document on line overlays/overlay-apiservice.yaml:5:
    Expected number of matched nodes to be 1, but was 2 (lines: test.yaml:184, upstream/components.yaml:181)
```

ggpaue committed:
remove extraneous test.yaml

ggpaue commented on addons/packages/metrics-server/0.6.1/package.yaml:
```diff
@@ -0,0 +1,143 @@
+    spec:
+      fetch:
+      - imgpkgBundle:
+          image: projects.registry.vmware.com/tce/metrics-server@sha256:4fa580c9b185d44f39847cd2347c8110b3a38d775e674ce77bc3b2569067f4c0
```
@seemiller should this image be manually pushed?

ggpaue replied:
thank you very much! @seemiller, I have pushed the changed SHA

ggpaue commented:
> I think the `test.yaml` file is extraneous, and it is causing an error with the overlays.
> 
> ```
> cd addons/packages/metrics-server/0.6.1/bundle/config
> ytt -f .
> ```
> 
> yields the error:
> 
> ```
> ytt: Error: Overlaying (in following order: overlays/overlay-apiservice.yaml, overlays/overlay-deployment.yaml, overlays/overlay-namespace.yaml, overlays/update-strategy-overlay.yaml):
>   Document on line overlays/overlay-apiservice.yaml:5:
>     Expected number of matched nodes to be 1, but was 2 (lines: test.yaml:184, upstream/components.yaml:181)
> ```

ah I added by mistake, now deleted it

ggpaue committed:
update pacakge sha
